standard/enums.md

```diff
@@ -26,7 +26,12 @@ enum_declaration
     ;
 
 enum_base
-    : ':' struct_type
+    : ':' integral_type
```

Comment:
These changes narrow down `enum_base` to avoid bad stuff happening. Text below has a small addition to describe the two alternatives.

Reply:
That seems fine